src/style.css

```
@@ -0,0 +1,74 @@
+.stars_item {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  min-height: 16px;
+  background-color: #fff;
+  background-repeat: no-repeat,
+    no-repeat,
+    no-repeat,
+    no-repeat,
+    no-repeat;
+  background-position: left top,
+    20px top,
+    40px top,
+    60px top,
+    80px top;
+}
+
+.stars__0 {
+  background-image: url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg);
+}
+
+.stars__1 {
+  background-image: url(./images/star-active.svg),
+    url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg);
+}
+
+.stars__2 {
+  background-image: url(./images/star-active.svg),
+    url(./images/star-active.svg),
+    url(./images/star.svg),
+    url(./images/star.svg),
+    url(./images/star.svg);
+}
+
+.stars__3 {
+  background-image: url(./images/star-active.svg),
+    url(./images/star-active.svg),
+    url(./images/star-active.svg),
+    url(./images/star.svg),
+    url(./images/star.svg);
+}
+
+.stars__4 {
+  background-image: url(./images/star-active.svg),
+    url(./images/star-active.svg),
+    url(./images/star-active.svg),
+    url(./images/star-active.svg),
+    url(./images/star.svg);
+}
+
+.stars__5 {
```

```suggestion
.stars__0,
.stars__1,
.stars__2,
.stars__3,
.stars__4,
.stars__5 {
```
Використовуй комбінатори селекторів, але з псевдокласом nth-child, як я описав у прикладі вище